Sources/PianoRoll/PianoRollNoteView.swift

```diff
@@ -99,8 +99,13 @@ struct PianoRollNoteView: View {
 
         // Main note body.
         ZStack(alignment: .trailing) {
-            Rectangle()
-                .foregroundColor(color.opacity((hovering || offset != .zero || lengthOffset != 0) ? 1.0 : 0.8))
+            ZStack(alignment: .leading) {
+                Rectangle()
+                    .foregroundColor(color.opacity((hovering || offset != .zero || lengthOffset != 0) ? 1.0 : 0.8))
+                Text(note.text ?? "")
```

Comment:
What's the behavior of this zstack if the text is longer than the rectangle?

Reply:
Text will be truncated

https://user-images.githubusercontent.com/4270232/198055419-2a3d2b2e-9dc3-44b0-bd4e-e2188098631d.mp4

Reply:
SwiftUI is smart enough to determine if text should be truncated based on the frame limit we set in https://github.com/AudioKit/PianoRoll/pull/7/files#diff-e2fedbc9a6fe669091b2ad9ea5f5e70164631e6de19aacf75ab318969b725707R116-R117